src/main/java/org/sagebionetworks/bridge/sdk/models/surveys/NumericalConstraints.java

```diff
@@ -0,0 +1,72 @@
+package org.sagebionetworks.bridge.sdk.models.surveys;
+
+import org.sagebionetworks.bridge.sdk.models.surveys.Unit;
+
+public class NumericalConstraints extends Constraints {
+
+    private Unit unit;
+    private Double minValue;
+    private Double maxValue;
+    private Double step;
+
+    public Unit getUnit() {
+        return unit;
+    }
+    public void setUnit(Unit unit) {
+        this.unit = unit;
+    }
+    public Double getMinValue() {
+        return minValue;
+    }
+    public void setMinValue(Double minValue) {
+        this.minValue = minValue;
+    }
+    public Double getMaxValue() {
+        return maxValue;
+    }
+    public void setMaxValue(Double maxValue) {
+        this.maxValue = maxValue;
+    }
+    public Double getStep() {
+        return step;
+    }
+    public void setStep(Double step) {
+        this.step = step;
+    }
+    
+    @Override
+    public int hashCode() {
```

Comment:
You'll want to regenerate (or delete) hashCode() and equals() to include Unit.

Is it also worth adding unit tests on equals() so we remember to update this?